### ISSUE TITLE
Entities are converted to symbols

### ISSUE THREAD

**Issue by cossssmin** [maizzle/framework#348]
```html
<div>&zwnj;</div>
<div>&nbsp;</div>
<p>&copy; 2020</p>
```

... is output as:

```html
<div></div>
<div> </div>
<p>© 2020</p>
```

This originated in [`htmlparser2@5.0.0`](https://github.com/fb55/htmlparser2/releases/tag/v5.0.0), which sets `decodeEntities` to `true` by default.

Disabling it in `config.js` does work:

```js
module.exports = {
  build: {
    posthtml: {
      options: {
        decodeEntities: false
      }
    }
  }
}
```

Ideally this should be fixed in `posthtml-parser` (ref https://github.com/posthtml/posthtml-parser/issues/51) by setting `decodeEntities: false` in its default options - adding this issue as reference.

**Comment by cossssmin** [comment]
v2.3.4 has a _temporary_ fix for this - you don't need to add the above config, but still entities get decoded into symbols if they're inside a layout or a component.

**Comment by cossssmin** [comment]
Fixed in [v2.4.0](https://github.com/maizzle/framework/releases/tag/v2.4.0)
